packages/bff/src/server.ts

```diff
@@ -97,16 +106,20 @@ const startServer = async (startTimeStamp: Date): Promise<void> => {
     },
   });
 
+  if (readinessProbeEnabled) {
+    startReadinessProbe(server, startTimeStamp);
```

Comment:
Readiness probe needs to be at the beginning of the program (as early as possible)